atat/src/ingress_manager.rs

```diff
@@ -208,7 +264,7 @@ where
         #[cfg(feature = "logging")]
         log::trace!("Receiving {} bytes", data.len());
         for byte in data {
-            match self.buf.push(*byte as char) {
+            match self.buf.push(*byte) {
```

Comment:
I think this could autoderef:

```suggestion
            match self.buf.push(byte) {
```

I'm not 100% sure though, so maybe test this first.

Reply:
Nope, :(
```
mismatched types
expected `u8`, found `&u8`
```